src/lib/env-vars.ts

```diff
@@ -94,7 +94,7 @@ export const SUPERVISOR_CONFIG_VAR_PROPERTIES: {
 	RESIN_SUPERVISOR_PERSISTENT_LOGGING: {
 		enum: ['false', 'true'],
 		description:
-			'Enable persistent logging. Only supported by resin-supervisor >= v7.15.0.',
+			'Enable persistent logging. Only supported by balena-supervisor >= v7.15.0.',
```

Comment:
I guess technically this should be "only supported by balena and resin supervisors >= v7.15.0" since there is no "balena" supervisor v 7.15.0